src/tests/Home.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import configureStore from 'redux-mock-store';
+import Home from '../components/Home';
+import '@testing-library/jest-dom/extend-expect';
+
+const mockStore = configureStore([]);
+
+describe('Home Component', () => {
+  let store;
+  beforeEach(() => {
+    store = mockStore({
+      weather: {
+        adminAreas: ['Area1', 'Area2'],
+        filteredAreas: [],
+      },
+    });
+  });
+
+  it('renders the Home component', () => {
+    render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <Home />
+        </MemoryRouter>
+      </Provider>,
+    );
+
+    expect(screen.getByText('TellTheClouds')).toBeInTheDocument();
```

Comment:
- [ ] 🚀 Impressive work so far on crafting such a beautiful `UI` for your capstone project with all the necessary features, as specified in the project requirements! 👏 Your attention to detail in creating the `UI` is truly commendable.

   - However, I'd like to point out that there are a few failing tests in your project. 😕 It's essential to address these issues to ensure that your project functions as intended. 🛠️

  - Taking the time to fix these failing tests is a crucial step in enhancing confidence in your project. When all the tests are passed, it provides a strong assurance that your project is operating as it should.

Below is a screenshot containing the details of the `failed` test:
<img width="1043" alt="Screenshot 2023-10-12 at 10 44 40" src="https://github.com/blessing-michael/Forecast-me/assets/107192076/fcc00458-2365-45fa-b824-4c9eb25dfeeb">

Happy coding! 😊👨‍💻🚀